cmd/server.go

```diff
@@ -79,6 +81,23 @@ func (s *Server) getRole(IP string) (string, error) {
 	return role, nil
 }
 
+func (s *Server) debugStoreHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
<3 I was just thinking about writing the same functionality while attempting to debug the store this weekend.